server/src/test/java/org/elasticsearch/cluster/routing/AllocationIdIT.java

```diff
@@ -66,6 +67,13 @@
 @ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.SUITE, numDataNodes = 0)
 public class AllocationIdIT extends ESIntegTestCase {
 
+    @Override
+    protected Settings nodeSettings(int nodeOrdinal) {
+        return Settings.builder().put(super.nodeSettings(nodeOrdinal))
+            .put(TestZenDiscovery.USE_ZEN2.getKey(), true)
```

Comment:
This test suite starts a three node cluster and then shuts down one node and then another, so it is a good test of the use of voting tombstones.